serving/installing-logging-metrics-traces.md

```diff
@@ -32,19 +20,19 @@ To configure and setup monitoring:
    public image [k8s.gcr.io/fluentd-elasticsearch:v2.0.4](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/fluentd-elasticsearch/fluentd-es-image).
    Or you can create a custom one and upload the image to a container registry
    which your cluster has read access to.
-1. Follow the instructions in
+2. Follow the instructions in
    ["Setting up a logging plugin"](setting-up-a-logging-plugin.md#Configuring)
    to configure the Elasticsearch components settings.
-1. Install Knative monitoring components by running the following command from the root directory of
+3. Install Knative monitoring components by running the following command from the root directory of
```

Comment:
Install (and uninstall) package questions: 

Is this just:

`kubectl apply -f https://github.com/knative/serving/releases/download/v0.2.2/monitoring.yaml`

Then the uninstall steps would simply be:

`kubectl delete -f https://github.com/knative/serving/releases/download/v0.2.2/monitoring.yaml`

Reply:
Moving questions about using *.yaml files to #613